dashboard/src/reducers/kube.ts

```diff
@@ -121,9 +121,9 @@ export const initialKinds = {
 export const initialState: IKubeState = {
   items: {},
   kinds: initialKinds,
+  // We book keep on subscriptions, keyed by the installed package ref,
+  // so that we can
```

Comment:
Maybe this comment is stripped out? I think I don't get the sentence

Reply:
Sorry - probably got distracted half way through writing it :P. Fixed.